Branch-SDK-TestBed/src/io/branch/branchandroiddemo/MainActivity.java

```diff
@@ -259,7 +259,7 @@ public void onClick(View view) {
                 // Define custom styel for the share sheet list view
                 //.setStyleResourceID(R.style.Share_Sheet_Style);
 
-                branchUniversalObject.showShareSheet(MainActivity.this, linkProperties, shareSheetStyle, new Branch.BranchLinkShareListener() {
+                branchUniversalObject.showShareSheet(MainActivity.this, linkProperties, shareSheetStyle, new Branch.BranchLinkShareListener2() {
```

Comment:
Can we keep this interface  unchanged? We can provide a separate builder method to set this call back so that existing partners are not affected with this change.